browser/src/view/components/Navbar/NavigationMenu.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { FiHome, FiLayers, FiSettings, FiLogOut, FiCircle } from 'react-icons/fi';
```

Comment:
We can import SVGs as SVGs directly from figma.  We don't need a library to manage this work for us.

Reply:
I need to double check the license for the Figma UI kit. I'm using a free version of it, so it's possible we can't actually use the icons. I'll look again, but that's why I'm using Feather instead via react-icons

Reply:
I'd actually rather we want in and snagged the icons out of the package than install another package lol.

Reply:
Sounds good! I think the Feather svg assets are free and easily available, so I'll look into that.